src/pages/Help.vue

```diff
@@ -105,7 +140,25 @@ export default {
     return {
       requests: [],
       selectedRequest: undefined,
-      isDialogOpen: false
+      isDialogOpen: false,
+      categories: [
+        { label: 'Kurierdienste', id: 1 },
+        { label: 'Warenleistungen', id: 2 },
+        { label: 'Bildung', id: 3 },
+        { label: 'Soziales & Gemeinschaft', id: 4 }
```

Comment:
Can we retrieve that by calling the API?
Nevertheless, i18n support is missing.

Reply:
Yes will do